### PR TITLE
docs: Mention `changelogCommitBodies` and add IntelliJ schema settings guide

### PR DESCRIPTION
**Pull request by mzdm**
## Description

https://github.com/invertase/melos/pull/606 `changelogCommitBodies` change is great, needed it badly so glad it exists, however is missing in the docs

I am regular IntelliJ user and having schema is a must have, so I added a small guide how to get it. 

Btw the https://github.com/blaugold/melos-code/pulls repo has some PRs to review

## Type of Change

- [ ] ✨ `feat` -- New feature (non-breaking change which adds functionality)
- [ ] 🛠️ `fix` -- Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ `!` -- Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 `refactor` -- Code refactor
- [ ] ✅ `ci` -- Build configuration change
- [x] 📝 `docs` -- Documentation
- [ ] 🗑️ `chore` -- Chore

### CODE REVIEW

File: docs/ide-support.mdx [hunk 1]
```diff
@@ -17,7 +17,16 @@ Melos will create an IntelliJ project module for each package in your Melos
 workspace, and will create flutter and dart Run configurations for your
 `main.dart`s and your package's test suite.
 
-![Generated Run configurations](/assets/intellij-run-configurations.png)
+<img src="/assets/intellij-run-configurations.png" alt="Generated Run configurations" height="600"/>
+
+### `melos.yaml`
+
+Autocompletion and validation of `melos.yaml` can be enabled in IntelliJ by going to `Settings` > `Languages & Frameworks` > `Schemas and DTDs` > `JSON Schema Mappings` and adding the Melos schema:
```
Comment (spydon) [inline docs/ide-support.mdx:24]: Can you wrap this so that the line length is <80?

**Review by spydon — COMMENTED** [review]
Looks good, just one small comment

File: docs/ide-support.mdx (hunk 1 above)
Comment (mzdm) [reply docs/ide-support.mdx]: Done 👍

File: docs/ide-support.mdx [hunk 2]
```diff
@@ -17,7 +17,17 @@ Melos will create an IntelliJ project module for each package in your Melos
 workspace, and will create flutter and dart Run configurations for your
 `main.dart`s and your package's test suite.
 
-![Generated Run configurations](/assets/intellij-run-configurations.png)
+<img src="/assets/intellij-run-configurations.png" alt="Generated Run configurations" height="600"/>
+
+### `melos.yaml`
+
+Autocompletion and validation of `melos.yaml` can be enabled in IntelliJ by going to
+`Settings` > `Languages & Frameworks` > `Schemas and DTDs` > `JSON Schema Mappings` and adding the Melos schema:
```
Comment (spydon) [inline docs/ide-support.mdx:25]: This is still longer than 80

Comment (mzdm) [reply docs/ide-support.mdx]: Ah, true

**Review by spydon — APPROVED** [review]
Thanks for your contribution!

**Comment by mzdm** [comment]
I've noticed the images does not get shown, I guess mdx does not support  directly. However the base images take a lot of space so I will resubmit smaller ones.

Is there a way by the way to build the docs locally? I did not find a way

**Comment by spydon** [comment]
> I've noticed the images does not get shown, I guess mdx does not support `img` directly. However the base images take a lot of space so I will resubmit smaller ones.
> 
> Is there a way by the way to build the docs locally? I did not find a way
> 
> @spydon

Ah, I missed this, you can preview your changes on https://melos.invertase.dev/~721

https://use.docs.page/previews#custom-domain

**Comment by mzdm** [comment]
> > I've noticed the images does not get shown, I guess mdx does not support `img` directly. However the base images take a lot of space so I will resubmit smaller ones.
> > Is there a way by the way to build the docs locally? I did not find a way
> > @spydon
> 
> Ah, I missed this, you can preview your changes on https://melos.invertase.dev/~721
> 
> https://use.docs.page/previews#custom-domain

Thx, I'll add it to Contributing.md guide

**Pull request merged**